shiny-ci-cd/actions/ci-cd.yml

```diff
@@ -6,46 +6,63 @@ name: CI-CD
 
 jobs:
   CI-CD:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      # we keep a matrix for convenience, but we would always want to run on one
+      # single OS and R version
+      matrix:
+        config:
+          - {os: ubuntu-20.04, r: 'release', rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
+
+    env:
+      RSPM: ${{ matrix.config.rspm }}
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
as in the renv case I'd add a comment here

Reply:
I don't see any comments regarding this section in any of the four .yml files in this folder, nor in the site.yaml. Not sure what you refer to.

As I already mentioned on https://github.com/miraisolutions/ShinyCICD/pull/1, we should view this together with the question there on how many / which comments we might want to add to these github actions workflows.

Reply:
I meant the comment in the review